### PR TITLE
Update to .NET Core 2, enable CI

### LINKED ISSUE #5: Add CI
We need to enable Travis and Appveyor.

### PR DESCRIPTION
**Pull request by ForNeVeR**
Closes #5.

Also chose to update to .NET Core 2.0, because it's easier to run tests and redistribute.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: csharp
+os: linux
+dist: trusty
```
Comment (Minoru) [inline .travis.yml]: IIRC Trusty is the default now, so this line isn't necessary. Or are you afraid that the build will break when Travis upgrades its images?

Comment (ForNeVeR) [reply .travis.yml]: > are you afraid that the build will break when Travis upgrades its images?
Yes.

**Comment by ForNeVeR** [comment]
Thanks for the review!

**Comment by rexim** [comment]
@ForNeVeR thank you for the PR

**Pull request merged**